mods/stairs/init.lua

```diff
@@ -4,17 +4,20 @@
 stairs = {}
 
```

Comment:
you could add here a table of things which should be copied from the recipeitem, e.g. {"alpha", "groups", …(and more from http://dev.minetest.net/register_node#Node_definition)} and set the things in the nodedef to the ones of the recipeitem instead of writing each one into the nodedef (https://github.com/minetest/minetest_game/pull/571/files#diff-d85e6749f6707aa8b7c089afb33ccdecR17)